lib/components/user/saved-trip-list.js

```diff
@@ -72,17 +69,53 @@ export default withLoggedInUserSupport(
   true
 )
 
+const TripHeader = styled.h3`
+  margin-top: 0px;
+`
+
+const PanelHeading = styled(Panel.Heading)`
+  background-color: white!important;
+`
+
+const PanelFooter = styled(Panel.Footer)`
+  background-color: white!important;
+  padding: 0px;
+  button {
+    border: 0px;
+    padding: 13px 0px;
+    width: 33.333%;
+  }
+
+  button:first-child {
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+
+  button:nth-child(2) {
+    border-radius: 0;
+    border-left: 1px solid #ddd;
+    border-right: 1px solid #ddd;
+  }
```

Comment:
Can this be combined with the generic `button` style block above?

Reply:
Which parts? In general, I think it's best to just keep it separate. I think these styles really only apply to the center button.